packages/api-reference/src/components/ApiReferenceLayout.vue

```diff
@@ -307,9 +309,11 @@ hideModels.value = props.configuration.hideModels ?? false
           </div>
         </template>
         <!-- REST API Client Overlay -->
+        <!-- Fonts are fetched by @scalar/api-reference already, we can safely set `withoutDefaultFonts: true` -->
         <ApiClientModal
           :parsedSpec="parsedSpec"
-          :proxyUrl="configuration?.proxy">
+          :proxyUrl="configuration?.proxy"
+          withoutDefaultFonts>
```

Comment:
Is this actually a prop on this component?

Reply:
Oops, probably not (yet). Need to take a look